admin/src/react-components/content-cdn.js

```diff
@@ -326,14 +326,14 @@ class ContentCDNComponent extends Component {
                 {hasValidWorkerDomain && (
                   <>
                     <li>
-                      In the Workers Dashboard click <b>Create Worker</b>.
+                      In the Workers Dashboard click <b>Create a Service</b>.
                     </li>
                     <li>
-                      Enter the name for the worker (at the top, above the script) as:
+                      Enter the name for the service (at the top, above the script) as:
```

Comment:
I'm surprised to see this change from `worker` to `service`. Is the thing that you're creating not a `worker`? (https://developers.cloudflare.com/workers/)

Reply:
Cloud flare called them services under worker now.   I believe because they want to push the serverless actions behind them.  
![image](https://user-images.githubusercontent.com/28871018/157510201-40407466-98e3-4af8-b746-cb6b118f6f49.png)